Issue50

vicgabal opened the pull request:
I have finished the gulpfile. It has tasks to compile and watrch html, jsx, scss files in FRONTEND/SOURCE and put them in FRONTEND/BUILD

I have deciced not to use for now the gulp.babel plugin

I want to test that it works. Does it look ok to you? 

The latest commits changing this file are all in my feature branch "issue50"

I am going to create now my index.html and app.jsx and work on the React components of the landing page

fedosejev commented on gulpfile.js:
```diff
@@ -0,0 +1,36 @@
+
+gulp.task('build-js', function() {
+  return browserify('./source/js/app.jsx')
+        .transform(reactify)
```
Update to `babelify`.

fedosejev commented on gulpfile.js:
```diff
@@ -0,0 +1,36 @@
+});
+
+gulp.task('watchers', function() {
+	gulp.watch('./frontend/**/*.jsx', ['build-js']);
```
You can use `*.{jsx, js}` to watch JSX _and_ JS.

fedosejev commented:
Create `default` task as well, so that you could simply run `gulp`.

vicgabal commented:
+ build paths corrected with ./
+ glob search for jsx and js grouped as .{jsx,js}
+ added a default task with an array of tasks that will run when comand gulp is executed in Terminal
+ since plugin reactify is deprecated, I have changed it for babelify: 
 + https://www.npmjs.com/package/babelify
 + npm install --save-dev babelify
 + npm install --save-dev babel-preset-es2015 babel-preset-react

vicgabal commented:
There is an app.jsx file in place but I get the error: 

Cannot find module '/Users/VictorG/gits/tiy-final-project/source/js/app.jsx' from '/Users/VictorG/gits/tiy-final-project'

vicgabal commented:
Error fixed. Missed to indicate that the parent folder is FRONTEND 
Also added React and React.DOM plugins, that were missed earlier
